src/main/webapp/app/home/home.scss

```diff
@@ -21,3 +21,67 @@ Main page styles
     background-size: contain;
   }
 }
+
+.p {
+  margin-top: 40px;
+  text-decoration: none;
+  color: black !important;
+}
+
+.box1 {
```

Comment:
`.box1`, `.box2` and `.box3`have the exact same css properties. Please create a `box`class and reuse it in the necessary different elements.

Reply:
done